feralinux_noconsole.spec

```
@@ -4,7 +4,7 @@ block_cipher = None
 a = Analysis(['fera.py'],
              pathex=['/media/sf_B_DRIVE/VISUALIZADOR/FERA'],
              binaries=[],
-             datas=[('./Imagens/fts4tutorial.png', '.'), ('./ListasDeBusca/*', './ListasDeBusca'), ('./Imagens/logoMini.ico', '.'), ('ffmpeg','.')],
+             datas=[('./ListasDeBusca/*', './ListasDeBusca'), ('./Imagens/*', './Imagens'), ('ffmpeg.exe','.'), ('whats_new.txt', '.')],
```

RuboCop (1.86.1)

[fatal] unexpected token tCOMMA
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)